activation/e2e/activation_test.go

```diff
@@ -187,8 +199,6 @@ func Test_BuilderWithMultipleClients(t *testing.T) {
 		},
 	).Times(totalAtxs)
 
-	verifier, err := activation.NewPostVerifier(cfg, logger.Named("verifier"))
-	require.NoError(t, err)
 	t.Cleanup(func() { assert.NoError(t, verifier.Close()) })
 	v := activation.NewValidator(db, poetDb, cfg, opts.Scrypt, verifier)
```

Comment:
NIT: these two should also be moved up with instantiation of the `verifier` (and we only need one `validator`/`v` 🙂 )

Reply:
nice catch